packages/coq/coq.8.7.0/opam

```diff
@@ -56,4 +56,4 @@ depends: [
   "camlp5"
   "num"
 ]
-available: [ocaml-version >= "4.02.3" & ocaml-version < "4.06.0"]
+available: [ocaml-version >= "4.02.3" & ocaml-version]
```

Comment:
This looks... funny? Is that even valid syntax? It's certainly not the way it used to look, with this additional `& ocaml-version`.

Reply:
No, I removed it.